grafana-plugin/src/components/IntegrationLogo/IntegrationLogoWithTitle.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+
+import { HorizontalGroup } from '@grafana/ui';
+
+import { Text } from 'components/Text/Text';
+
+import { IntegrationLogo, IntegrationLogoProps } from './IntegrationLogo';
```

Comment:
Not related to this PR, but what if we start sticking to just absolute paths instead? Might be a good start with new PRs

Reply:
It's the eslint that should take care of that and currently our setup is:
```
'rulesdir/no-relative-import-paths': ['error', { allowSameFolder: true }],
```

So basically it allows relative path as soon as you import stuff from the same folder. So it allows: `./Dir/AnotherDir` or `./file.ts` but doesn't allow `../AnotherDir/file.ts`. And tbh that's the config I find useful and I've always used. Because if you're working on deeply nested directory like `src/dir/dir2/dir3/dir4/dir5/myFile` and you want to import `myFile2` from your current directory I think it's easier and more readable to have `./myFile2` rather than `src/dir/dir2/dir3/dir4/dir5/myFile2`

Reply:
Gotcha, might be a good topic to discuss in one of the frontend meetings. Didn't know it's defined as an eslint rule, I thought that's the default behavior on prettier formatting.